eo-maven-plugin/src/test/java/org/eolang/maven/WeHaveCargo.java

```diff
@@ -35,12 +35,13 @@
 
 /**
  * JUnit's extension to skip a test if there is no cargo.
- * Cargo is a build system for rust so it is required for rust inserts.
+ * Cargo is a build system for rust. so it is required for rust inserts.
```

Comment:
@levBagryansky typo. "So" instead of "so"

Reply:
@maxonfjvipon Indeed